pilz_robot_programming/src/pilz_robot_programming/commands.py

```diff
@@ -185,7 +185,13 @@ def __init__(self, goal=None, planning_group=_DEFAULT_PLANNING_GROUP, target_lin
         # Needs to be set by derived classes
         self._planner_id = None
 
-        self._goal = goal
+        try:
+            if isinstance(goal, str):
+                raise TypeError()
```

Comment:
Should probably throw an `NotImplementedError` in the furture, but didn't want to introduce an api break